textgrid/textgrid.py

```diff
@@ -297,7 +297,7 @@ def __contains__(self, other):
             return self.minTime <= other <= self.maxTime
 
     def bounds(self):
-        return (self.minTime, self.maxTime)
+        return self.minTime, self.maxTime
```

Comment:
Can you easily un-revert the deleted parens? I love parens around tuples.

Reply:
I reverted the removal.